src/dos/program_mousectl.cpp

```diff
@@ -544,10 +544,11 @@ void MOUSECTL::AddMessages()
 	        "  -s -sx -sy     sets sensitivity / for x axis / for y axis, from -999 to +999\n"
 	        "  -r             sets minimum mouse sampling rate\n"
 	        "  -on -off       enables or disables mouse on the given interface\n"
-	        "  -reset         restores all mouse settings from the configuration file\n"
+	        "  -reset         restores default mouse settings\n"
 	        "\n"
 	        "Notes:\n"
 	        "  If sensitivity or rate is omitted, it is reset to default value.\n"
+	        "  Sensitivity set in the configuration file works independently.\n"
```

Comment:
I think this needs more clarification. We should make it clear here that the `mouse_sensitivity` settings acts as a global scale factor, and then per mouse interface sensitivity settings are applied on top of that.

Reply:
OK, added more information.